### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by mresofe** [community/community#45773] [Other Feature Feedback, Questions, & Ideas]
I need some help in my java please 🙏

**Comment by SakibSumon** [comment]
what help do you need?

**Reply by mresofe** [reply]
Hello can I send i picture

**Reply by mresofe** [reply]
![image](https://user-images.githubusercontent.com/123942524/215546800-9781f73b-4b2e-4d4c-bb93-b110827cdc04.png)

**Comment by mresofe** [comment]
Hello ty

בתאריך יום ב׳, 30 בינו׳ 2023 ב-13:46 מאת Sakib Rahman Sumon <
***@***.***>:
